Python/sort/Matrix_Sort.py

```diff
@@ -0,0 +1,57 @@
+def sort_matrix(mat, size):
```

Comment:
Add docstring to explain how the function works.

Reply:
Added the docstring and checked for PEP8 requirements too. @atarax665